src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,71 @@
+package duke;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * Duke class which is the main entry point when starting up the application
+ */
+public class Duke {
+
+    private Ui ui;
+    private TaskList taskList;
+    private Storage storage;
+    private Parser parser;
+
+    /**
+     * Constructor for Duke class
+     */
+    public Duke(String path, String file_dir) {
```

Comment:
```suggestion
    public Duke(String path, String fileDir) {
```

variable names should be in camelCase

Reply:
same for `file_dir` in Storage.java